src/components/cohortStudent/index.js

```diff
@@ -0,0 +1,21 @@
+import ProfileCircle from "../profileCircle"
+import "./style.css"
+
+
+const CohortStudent = ({firstName, lastName}) => {
+
+    const userInitials = `${firstName[0]}${lastName[0]}`
```

Comment:
Let's add a check before this line that `firstName` and `lastName` have a length greater than 0. Otherwise this will error

Reply:
Enclosed the component with an if statement to check if the first name and last name have length greater than zero.